client/landing/subscriptions/components/site-list/styles.scss

```diff
@@ -90,14 +90,29 @@ $max-list-width: 1300px;
 
 		.title-box,
 		.email-frequency,
-		.date {
+		.date,
+		.new-posts,
+		.new-comments {
 			font-weight: 400;
 			font-size: $font-body-small;
 			line-height: 20px;
 			letter-spacing: -0.15px;
 			color: $studio-gray-60;
 		}
 
+		.new-posts,
+		.new-comments {
+			.green {
+				fill: var(--color-success);
```

Comment:
Looking at Figma (inDLaEQV8jJ21O4WXawIsJ-fi-712_18319), I see `Green 60` (`#007017`) used there - instead of `#008a20`.

Reply:
I guess we should follow the colors in the Figma design. My concern is the green color doesn't match the rest of the ✓ icons we have in Calypso, which have --color-success on it.

Reply:
As per the discussion here, p1683024291119279-slack-C02TCEHP3HA, we will change the design to Green 50 to maintain consistency with the rest of the application.